anndata/core/alignedmapping.py

```diff
@@ -94,7 +91,7 @@ def isview(self) -> bool:
         pass
 
     @property
-    def parent(self) -> "AnnData":
+    def parent(self) -> Union["anndata.AnnData", "raw.Raw"]:
```

Comment:
I’m unsure what here can have Raw as a parent, and how we should handle this going forward.

Using a AnnData base that supports all the interfaces used by AnnData, Raw, and so on?

Reply:
Hmm. I can't follow the link from the edit due to the rebase, but I assume this includes `varm`?

Reply:
Yes, Raw can have `varm` which is an `AxisArrays`, which means that `parent` can be a `Raw`.

Reply:
Ah, I think I didn't quite get what you were asking. 

Yeah, I was thinking an `AnnDataBase` or `AbstractAnnData` class would be good. It's a little weird how methods would get split up since raw only has the `var` ones, but it would be good to reuse code.